src/loc.rs

```diff
@@ -220,11 +290,18 @@ impl Loc for CppCode {
         let start = init(node, stats, is_func_space, is_unit);
 
         match node.kind_id().into() {
-            RawStringLiteral | StringLiteral | ExpressionStatement | CompoundStatement
-            | LabeledStatement | DeclarationList | FieldDeclarationList | TranslationUnit => {}
+            RawStringLiteral | StringLiteral | DeclarationList | FieldDeclarationList
+            | TranslationUnit => {}
             Comment => {
                 stats.comment_lines.insert(start);
             }
+            WhileStatement | SwitchStatement | CaseStatement | IfStatement | ForStatement
+            | ReturnStatement | BreakStatement | ContinueStatement | GotoStatement
+            | ThrowStatement | TryStatement | ExpressionStatement | LabeledStatement
+            | StatementIdentifier => {
+                *stats.logical_lines.entry(start).or_insert(0) += 1;
```

Comment:
So for example with `for (;;) break;` the lloc will be 2.
Is it what you're expecting ?
If yes please add documentation and a test would be nice too.

Reply:
Can we consider `;;` as statements? If yes, lloc should be 2.

Anyway, this PR is not ready yet. I need to work on that more (unit tests + other checks)

Reply:
It's already 2 because we've a ForStmt and a BreakStmt

Reply:
> It's already 2 because we've a ForStmt and a BreakStmt.

True, but can we consider this output correct? 

> So finally this metric is number of statements, correct ?

Exactly

> If it's the case the lloc name seems to be inappropriate.

lloc means `Logical Lines of Code` and according to Wikipedia and other papers, logical lines are statements. That's why I've maintained that name.